shells/planner-shell/single-user-context.js

```diff
@@ -192,15 +192,14 @@ export const SingleUserContext = class {
     // TODO(sjmiles): cache and return promises in case of re-entrancy
     let promise = this.pendingStores[id];
     if (!promise) {
-      promise = new Promise(async (resolve) => {
-        const store = await context.findStoreById(id);
-        if (store) {
-          resolve(store);
-        } else {
-          const store = await context.createStore(type, name, id, tags);
-          resolve(store);
+      // eslint-disable-next-line no-async-promise-executor
```

Comment:
I don't think you need this line now.

Reply:
Fwiw, `planner-shell` is old and not maintained. 

I don't know if the shell even works, but I do know this context code is out-dated.

Reply:
done